libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/service/MapboxTripService.kt

```diff
@@ -20,23 +22,20 @@ import kotlinx.coroutines.channels.ReceiveChannel
  * @param tripNotification provide contract to communicate with notification
  * @param initializeLambda called when [TripService] has started
  * @param terminateLambda called when [TripService] has stopped
+ * @param logger interface for logging any events
  */
 class MapboxTripService(
     private val tripNotification: TripNotification,
     private val initializeLambda: () -> Unit,
-    private val terminateLambda: () -> Unit
+    private val terminateLambda: () -> Unit,
+    private val logger: Logger
 ) : TripService {
 
-    companion object {
-        private var notificationDataChannel = Channel<MapboxNotificationData>(1)
-        internal fun getNotificationDataChannel(): ReceiveChannel<MapboxNotificationData> =
-            notificationDataChannel
-    }
```

Comment:
For consistency we should apply this codestyle change across the SDK codebase (it can be in a follow up PR). Refs. https://github.com/mapbox/mapbox-navigation-android/pull/2774#discussion_r412299701 cc @kmadsen

Reply:
Agree, lets apply one general codestyle across the SDK. But lets do it in a separate PR.